tests/test_normalization.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+import os
+from tests import MODEL_DIR, INPUT_DIR, OUTPUT_DIR
+
+from sample_annotator.ontology.package_checklist_normalizer import normalize_package
+from sample_annotator.report_model import AnnotationReport
+from sample_annotator.ontology.triad_normalizer import normalize_triad_slot
+
+"""Test the module can be imported."""
+
+import unittest
+
+
+class TestNormalizastions(unittest.TestCase):
+    """annotation test."""
+
+    def test_env_package_success(self):
+        normalized = normalize_package("MIGS/MIMS/MIMARKS.host-associated")
+        # print(normalized)
+        assert normalized == "host-associated"
+
+    def test_env_package_failure(self):
+        normalized = normalize_package("made up package")
+        assert normalized == ""
```

Comment:
consider return None if cannot be found

Reply:
How can I hint that the return type is a string _or_ None?

Reply:
done